toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -31,6 +31,13 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+var (
+	// Image customizer input validation errors
+	ErrInvalidOutputFormat            = NewImageCustomizerError("Customizer:InvalidOutputFormat", "invalid output image format")
+	ErrCannotGenerateOutputFormat     = NewImageCustomizerError("Customizer:CannotGenerateOutputFormat", "cannot generate output format from input format")
+	ErrCannotCustomizePartitionsOnIso = NewImageCustomizerError("Customizer:CannotCustomizePartitionsOnIso", "cannot customize partitions when input is ISO")
+)
+
```

Comment:
look through this file and implement error objects in the same pattern as the rest of this PR, but only for the errors that are for a function that does not return a named error objects already.

For example, the error below should have an error object InvalidImageConfig that would be used instead.
`if err != nil {
		return fmt.Errorf("invalid image config:\n%w", err)
}`

Update any test case error assertions to match the new error message if needed.